src/uu/chown/src/chown.rs

```diff
@@ -166,7 +166,12 @@ pub fn uu_app() -> App<'static, 'static> {
 }
 
 fn parse_spec(spec: &str) -> UResult<(Option<u32>, Option<u32>)> {
-    let args = spec.split_terminator(':').collect::<Vec<_>>();
+    let args = if spec.contains('.') {
+        spec.split_terminator('.')
+    } else {
+        spec.split_terminator(':')
+    }
+    .collect::<Vec<_>>();
```

Comment:
```suggestion
    let args = spec.split_terminator(&[':', '.'][..]).collect::<Vec<_>>();
```

Reply:
Cool, thanks :)
I was sure there would be a better way! thanks